examples/combined_ODNP.py

```diff
@@ -142,108 +127,63 @@
         time.sleep(5)
         power_settings_dBm[j] = p.get_power_setting()
         time_axis_coords[j + 1]["start_times"] = time.time()
-        logger.debug("gonna run the Ep for this power")
         run_spin_echo(
-            nScans=nScans,
+            nScans=parser_dict['nScans'],
             indirect_idx=j + 1,
             indirect_len=len(powers) + 1,
-            adcOffset=adcOffset,
-            carrierFreq_MHz=carrierFreq_MHz,
+            adcOffset=parser_dict['adc_offset'],
+            carrierFreq_MHz=parser_dict['carrierFreq_MHz'],
             nPoints=nPoints,
-            nEchoes=nEchoes,
-            p90_us=p90_us,
-            repetition=repetition_us,
-            tau_us=tau_us,
-            SW_kHz=SW_kHz,
-            output_name=output_name,
+            nEchoes=parser_dict['nEchoes'],
+            p90_us=parser_dict['p90_us'],
+            repetition=parser_dict['repetition_us'],
+            tau_us=parser_dict['tau_us'],
+            SW_kHz=parser_dict['SW_kHz'],
+            output_name=filename,
             ret_data=DNP_data,
         )
         time_axis_coords[j + 1]["stop_times"] = time.time()
 DNP_data.set_prop("stop_time", time.time())
-DNP_data.set_prop("postproc_type", Ep_postproc)
-acq_params = {
-    j: eval(j)
-    for j in dir()
-    if j
-    in [
-        "adcOffset",
-        "carrierFreq_MHz",
-        "amplitude",
-        "nScans",
-        "nEchoes",
-        "p90_us",
-        "deadtime_us",
-        "repetition_us",
-        "SW_kHz",
-        "nPoints",
-        "deblank_us",
-        "tau_us",
-        "nPhaseSteps",
-        "MWfreq",
-        "power_settings_dBm",
-    ]
-}
-DNP_data.set_prop("acq_params", acq_params)
-DNP_data.name("enhancement")
-DNP_data.chunk("t", ["ph1", "t2"], [4, -1])
+DNP_data.set_prop("postproc_type", "spincore_ODNP_v3")
+DNP_data.set_prop("acq_params", parser_dict.asdict())
+DNP_data.name(parser_dict['type'])
+DNP_data.chunk("t", ["ph1", "t2"], [len(Ep_ph1_cyc), -1])
 DNP_data.setaxis("ph1", Ep_ph1_cyc / 4)
+DNP_data.setaxis('nScans',r_[0:parser_dict['nScans']])
```

Comment:
this was missing but was added in when addressing the thermal scan issue

Reply:
This was done in [this task](https://www.wrike.com/open.htm?id=937064458) so I keep things the same for reproducibility